analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -198,54 +207,15 @@ class SpdxDocumentFile(
         val packages: MutableSet<Package> = mutableSetOf()
 
         val project = if (projectPackage != null) {
-            val scopes = listOfNotNull(
-                Scope(
-                    name = DEFAULT_SCOPE_NAME,
-                    dependencies = getDependencies(projectPackage, spdxDocument, packages, workingDir)
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.BUILD_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.DEV_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.OPTIONAL_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.PROVIDED_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.RUNTIME_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                ),
-                createScope(
-                    spdxDocument,
-                    projectPackage,
-                    SpdxRelationship.Type.TEST_DEPENDENCY_OF,
-                    packages,
-                    workingDir
-                )
-            ).toSortedSet()
+
+            val scopes = SPDX_SCOPE_RELATIONSHIPS.mapNotNull { type ->
+                createScope(spdxDocument, projectPackage, type, packages, workingDir)
+            }.toSortedSet()
```

Comment:
Drop `.toSortedSet()` here in favor of using `.mapNotNullTo(sortedSetOf())` above.